### PR TITLE
Glib2 regex match simple

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

**Comment by kou** [comment]
How about implementing `GLib::Regex.match` by the following?:

```ruby
class << self
  def match(pattern, string, options={})
    new(pattern, options).match(string, options)
  end
end
```

I don't want to use `_simple` suffix in method name. Because it's not Rubyish. Rubyish API don't provide simple feature method and full feature method. You can use simple feature by not providing additional argument.

**Comment by cedlemo** [comment]
Of course, I didn't thought about that.

**Comment by cedlemo** [comment]
Done

File: glib2/lib/glib2/regex.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+module GLib
+  class Regex
+    class << self
+      def match(pattern, string, options={})
```
Comment (kou) [inline glib2/lib/glib2/regex.rb]: Ah, it should have `?` suffix. Because it's a predicate method.

File: glib2/test/test-regex.rb [hunk 2]
```diff
@@ -194,4 +194,15 @@ def test_match_flags
   def test_regex_escape_string
     assert_equal("a\\.b\\*c", GLib::Regex.escape_string("a.b*c"))
   end
+
+  sub_test_case "match_simple" do
+    test "true" do
+      matched = GLib::Regex.match("to", "tatota")
+      assert(matched)
```
Comment (kou) [inline glib2/test/test-regex.rb]: Use the following style:

```ruby
assert do
  GLib::Regex.match?("to", "tatota")
end
```

**Comment by cedlemo** [comment]
Done :)

**Comment by kou** [comment]
Thanks!

**Pull request merged**